src/_P073_7DGT.ino

```diff
@@ -1139,6 +1139,7 @@ bool p073_plugin_write_7dn(struct EventStruct *event, const String& text) {
   switch (P073_data->displayModel) {
     case P073_TM1637_4DGTCOLON: {
       if ((event->Par1 > -1000) && (event->Par1 < 10000)) {
+        // FIXME TD-er: Why call round on an int?
         P073_data->FillBufferWithNumber(String(round(event->Par1)));
```

Comment:
It was previously `String(int(event->Par1))`, and that shouldn't have needed the `int()` part either.

When replacing that with `text.c_str()`, like all other cases of this switch use, this case's code can actually be removed as it is a duplicate of the next case, so a fall-through would be appropriate. 👍

Reply:
If you are modifying this file, there are also a couple of strings without the `F()` macro applied, around line 1037 and further, (12 in total).

Reply:
I can also remove the file from this PR, so you can fix it yourself :)

Reply:
In that case it would probably also be feasible to refactor most code into separate `P073_data_struct` files, as currently all code is included in the `ESPEasy.ino` file.